src/main/java/io/microconfig/osdf/chaos/ChaosExperiment.java

```diff
@@ -52,20 +66,22 @@ public void check() {
 
     public void run() {
         announce("Launch of chaos");
+
         IntStream.range(0, durationParams.getStagesNum()).forEach(stage -> {
             announce("Launch of chaos stage " + (stage + 1));
             Set<Chaos> currentStageChaosSet = chaosSet.stream().map(list -> list.get(stage)).collect(toUnmodifiableSet());
             currentStageChaosSet.forEach(Chaos::run);
             try {
```

Comment:
try to follow single level abstraction principle
this can be a separate method

Reply:
divided into several methods